content/docs/command-reference/fetch.md

```diff
@@ -136,8 +136,7 @@ $ cd example-get-started
 
 </details>
 
-The workspace looks almost like in this
-[pipeline setup](/doc/tutorials/pipelines):
+The workspace looks almost like in this [pipeline setup](/doc/tutorials/dags):
```

Comment:
/doc/tutorials/pipelines no longer exists. Find a better doc to link to (maybe /doc/start/data-pipelines) or remove.

Reply:
I don't think the link is needed here. There is no direct relevance to pipeline here, it's just explaining the command and talking about the similarity in the project.